app/views/teacher_training_adviser/steps/completed.html.erb

```diff
@@ -31,7 +31,7 @@
 
         <p>You need to reply to the email to be assigned an adviser.</p>
 
-        <p>It may take us longer to respond to you during the holiday period until early January. We’ll be in touch as soon as we can, thank you for your patience.</p>
+        <p>It may take us longer to respond to you because we're getting a lot of requests for advisers. We’ll be in touch as soon as we can. Thank you for your patience.</p>
```

Comment:
```suggestion
        <p>It may take us longer to respond to you because we're getting a lot of requests for advisers at the moment. We’ll be in touch as soon as we can. Thank you for your patience.</p>
```